hw/xbox/xid.c

```diff
@@ -235,12 +252,24 @@ static void usb_xid_handle_control(USBDevice *dev, USBPacket *p,
         }
         break;
     case ClassInterfaceOutRequest | HID_SET_REPORT:
-        DPRINTF("xid SET_REPORT %x\n", value);
-        assert(false);
+        DPRINTF("xid SET_REPORT 0x%x\n", value);
+        if (value == 0x200) { /* output */
+            /* Read length, then the entire packet */
+            memcpy(&s->out_state, data, sizeof(s->out_state));
+            assert(s->out_state.length == sizeof(s->out_state));
+            assert(s->out_state.length <= length);
+            //FIXME: Check actuator endianess
+            printf("Set rumble power to 0x%x, 0x%x\n",
```

Comment:
I decided for printf so we actually have some form of output. If we had a proper input layer we could set the rumble power, however, for devices such as mice and keyboards there should be another indicator. This is as simple as it gets.